allauth/socialaccount/templatetags/socialaccount.py

```diff
@@ -52,7 +55,10 @@ def provider_login_url(parser, token):
 
 class ProvidersMediaJSNode(template.Node):
     def render(self, context):
-        request = context['request']
```

Comment:
This introduces some duplication.. how about adding a `get_request_from_context` to `allauth/compat.py` ?